core/domain/skill_services_test.py

```diff
@@ -1010,17 +1010,6 @@ def test_filter_skills_by_mastery(self):
         self.assertEqual(arranged_filtered_skill_ids, self.SKILL_IDS)
 
 
-# TODO(lilithxxx): Remove this mock class and tests for the mock skill
-# migrations once the actual functions are implemented.
-# See issue: https://github.com/oppia/oppia/issues/7009.
```

Comment:
I don't see #7009 resolved, so why are we removing this?

Reply:
We can close #7009 now because, it has been solved.
Also the functions mentioned in the issue are added in this migration making the mock class redundant.
So I think It's fine to remove this TODO.